src/rust/src/orchard_ffi.rs

```diff
@@ -74,6 +74,15 @@ pub extern "C" fn orchard_bundle_serialize(
     }
 }
 
+#[no_mangle]
+pub extern "C" fn orchard_bundle_value_balance(bundle: *const Bundle<Authorized, Amount>) -> i64 {
```

Comment:
Why are some FFI functions declared with ABI "C", and some with ABI "system"?

Reply:
Most are declared as `C`. The `system` ones were added by Nikolay when integrating ZIP 221, and I just never noticed them. We can clean this up at some point.

Reply:
Clean *them up; the `C` ABI is the one we want to use here.